CMP1903M Object Oriented Programming A01 2223/CMP1903M A01 2223/CMP1903M A01 2223/Testing.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CMP1903M_A01_2223
+{
+    class Testing
+    {
+        public void Test()
+        {
+            int[] cardArray = { };
+            
+            //Instantiates objects
+            Card c1 = new Card();
+            Pack p1 = new Pack();
+
+            int[] cards = p1.cardPack.ToArray();

```

Comment:
This is an empty array because the array initialized inside the Pack class holds nothing